Makefile

```
@@ -1,25 +1,55 @@
-ifndef TAG
-	TAG = local
+REGISTRY=cfaprdbatchcr.azurecr.io/
+IMAGE_NAME=cfa-epinow2-pipeline
+BRANCH=$(shell git branch --show-current)
+ifeq ($(BRANCH), 'main')
+TAG=latest
+else
+TAG=$(BRANCH)
 endif
 
-IMAGE_NAME=cfa-epinow2-pipeline
+CONFIG=test.json
+JOB=batch-test
+
 
 deps:
 	docker build -t $(REGISTRY)$(IMAGE_NAME)-dependencies:$(TAG) -f Dockerfile-dependencies
 
 pull:
+	az acr login --name 'cfaprdbatchcr'
 	docker pull $(REGISTRY)$(IMAGE_NAME)-dependencies:$(TAG)
+	docker pull $(REGISTRY)$(IMAGE_NAME):test-$(TAG)
 
 build:
-	docker build -t $(REGISTRY)$(IMAGE_NAME):$(TAG) \
+	docker build -t $(REGISTRY)$(IMAGE_NAME):test-$(TAG) \
 		--build-arg TAG=$(TAG) -f Dockerfile .
 
 tag:
 	docker tag $(IMAGE_NAME):$(TAG) $(REGISTRY)$(IMAGE_NAME):$(TAG)
 
+config:
+	gh workflow run \
+	  -R cdcgov/cfa-config-generator run-workload.yaml  \
+	  -f disease=all \
+	  -f state=all
+
+run-batch:
+	docker build -f Dockerfile-batch -t batch . --no-cache
+	docker run --rm  \
+	--env-file .env \
+	-it \
+	batch python job.py "cfa-epinow2-edit-azure-flow" "$(JOB)"
+
+run:
+	docker run --mount type=bind,source=$(PWD),target=/cfa-epinow2-pipeline -it \
+	--env-file .env \
+	--rm $(REGISTRY)$(IMAGE_NAME):test-$(TAG) \
+	Rscript -e "CFAEpiNow2Pipeline::orchestrate_pipeline('$(CONFIG)', config_container = 'rt-epinow2-config', input_dir = '/cfa-epinow2-pipeline/input', output_dir = '/cfa-epinow2-pipeline', output_container = 'zs-test-pipeline-update')"
```

```suggestion
	Rscript -e "CFAEpiNow2Pipeline::orchestrate_pipeline('$(CONFIG)', config_container = 'rt-epinow2-config', input_dir = '/cfa-epinow2-pipeline/input', output_dir = '/', output_container = 'zs-test-pipeline-update')"
```
Use the same output dir as for the `run-batch` command. This way, the file paths in the metadata will always be the same as the those created by the batch command, which is desirable for maintaining consistency in output folder structure. However, this would require changing the bind mount above to mount to the container's root, instead of `/cfa-epinow2-pipeline`. 

What do you think?

---

I like the point about consistency, but I think we should follow the pattern of this one, not `run-batch`. We want the local run to write into the bind mount, which this edit would prevent.

---

But what if we change the bind mount to be `/` inside the container for this make command?

---

I think in `/mnt/` would be more traditional?

---

db06bab